### ISSUE TITLE
Error: Looks like multiple versions of prosemirror-model were loaded 

### ISSUE THREAD

**Issue by EmilBayt** [ueberdosis/tiptap#2129]
### What’s the bug you are facing?

When I am pressing enter I am getting this error `can not convert <> to a fragment (looks like multiple versions of prosemirror-model were loaded)`. I know that most probably you will refer to this issue [#577](https://github.com/ueberdosis/tiptap/issues/577) but this solution does not work for me. I started getting this error as soon as I upgraded tiptap to version 2. And I think I was following the instructions in this [Guide](https://tiptap.dev/overview/upgrade-guide), which means that before starting I deleted the old version of tiptap. What else I can do to solve this problem?

### How can we reproduce the bug on our side?

press enter and receive error `can not convert <> to a fragment (looks like multiple versions of prosemirror-model were loaded)`

### What did you expect to happen?

I expected to be able to write the new line after pressing enter

### Did you update your dependencies?

- [X] Yes, I’ve updated my dependencies to use the latest version of all packages.

### Are you sponsoring us?

- [ ] Yes, I’m a sponsor. 💖

**Comment by philippkuehn** [comment]
It’s definitely a duplicate of #577

Please look at your lock files and search for the source of your duplicated versions of prosemirror-model.

**Comment by EmilBayt** [comment]
surprisingly i was able to solve problem by deleting 
`  "resolutions": {
    "prosemirror-model": "1.9.1"
  }
I tried that before and it did not work, but now looks fine, thanks and sorry for bothering 🙂
